porcupine/menubar.py

```diff
@@ -358,6 +358,11 @@ def setup() -> None:
     set_enabled_based_on_tab(path, (lambda tab: isinstance(tab, tabs.FileTab)))
 
 
+def add_filetype_menuitem(label: str, func: Callable[[tabs.FileTab], object], tk_var: tkinter.StringVar) -> None:
+    get_menu('Filetypes').add_radiobutton(label=f"{label}",
```

Comment:
- You can simply do this in `plugins/filetypes.py`. It is the only file that knows anything about filetypes, and we don't need to change that.
- Nit: `f"{label}"` is unnecessary and same as just `label`.